modules/ROOT/pages/slsa.adoc

```diff
@@ -0,0 +1,156 @@
+
+a| https://slsa.dev/spec/v1.0/verifying-artifacts#step-1-check-slsa-build-level[Verifying Artifacts (v1.0)]
+a| xref:ec-policies:ROOT:release_policy.adoc#slsa_provenance_available__attestation_predicate_type_accepted[Expected attestation predicate type found]
+a| SLSA suggests that the predicate type in the artifact's provenance should be `https://slsa.dev/provenance/v1`. This rule confirms that requirement,
+and will fail if the predicate type is found to be any other value.
+
+a| https://slsa.dev/spec/v1.0/verifying-artifacts#step-1-check-slsa-build-level[Verifying Artifacts (v1.0)]
+a| xref:ec-policies:ROOT:release_policy.adoc#provenance_materials__git_clone_source_matches_provenance[Git clone source matches materials provenance]
+a| This is a good sanity check related to artifact verification that isn't specifically mentioned in the SLSA guidelines. In RHTAP build pipelines the
+git clone task provides information about the git repo and the commit sha used. This rule confirms that the url and the commit sha from the git clone
+task match what appears in the "materials" provenance.
+
+|===
+
+=== SLSA requirements not explicitly verified by Enterprise Contract
+
+There are numerous SLSA Build requirements that are met by Red Hat Trusted
+Application Pipeline, but not explicitly verifiable with an Enterprise Contract
+policy rule. We won't go into depth in this document, but for example:
+
+[cols="1,3"]
+|===
+|SLSA reference|RHTAP response
+
+a| https://slsa.dev/spec/v1.0/levels#build-l2-hosted-build-platform[Build L2: Hosted build platform]
+a| Due to its design, its architecture, and operational management, the Red Hat Trusted Application
+Pipeline, meets these requirements implicitly. And, as we've seen above, the Enterprise Contract
+can be used to verify the authenticity of the signed provenances.
+
+a| https://slsa.dev/spec/v1.0/levels#build-l3-hardened-builds[Build L3: Hardened builds]
+a| For example, the RHTAP's Tekton environment prevents different build pipelines from influencing each
+other, and has secure access polices and management practices for signing secrets and other credentials.
+
+|===
+
+////
+In future there may be some RHTAP system level docs addressing this in more detail.
+Linking to a general system overview of RHTAP would also be useful here also I think.
+////
+
+== Notes and caveats
+
+Because EC was designed initially for use with RHTAP, and RHTAP uses Tekton
+CI/CD, many of the specific rules mentioned above are currently Tekton specific
+in that they use details about the Tekton pipeline run, which are currently
+made available in the SLSA attestation created by Tekton Chains.
+
+This is not a limitation of EC itself, but rather just an attribute of the
+xref:ec-policies:ROOT:release_policy.adoc[policies created for RHTAP] in their
+current state. In future we're aiming to extract the Tekton specific rules from
+the general SLSA rules to make the rules more generally useful. It's expected
```

Comment:
I dunno. Just so as not to imply that they're not useful atm:

```suggestion
the general SLSA rules to make the rules more generally applicable. It's expected
```

Reply:
Just to match the speaker of the sentence before.

```suggestion
the general SLSA rules to make the rules more generally useful. We expect
```

Reply:
I missed the second one intially, but added in commit 2af8b8e .